docs/change_log.rst

```diff
@@ -1,13 +1,14 @@
-.. This Source Code Form is subject to the terms of the Mozilla Public
-.. License, v. 2.0. If a copy of the MPL was not distributed with this
-.. file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 .. _change-log:
 
 ======================
 Change Log
 ======================
 
+v1.1.0 - Jun 2, 2023
```

Comment:
Worth tweaking the date, so that the commit/merge/squash history lines up with the release date

```suggestion
v1.1.0 - Jul 12, 2023
```